api/graphql/schema.graphql

```diff
@@ -164,6 +196,7 @@ type Query {
     cmoAssets: [CmoAsset]! @all @guard @can(ability: "viewAny")
     department(id: ID! @eq): Department @find @guard @can(ability: "viewAny")
     departments: [Department]! @all @guard @can(ability: "view")
+    candidateFilterSearch
```

Comment:
`php artisan lighthouse:validate-schema` returns an error:

![Screen Shot 2021-09-20 at 11 20 37](https://user-images.githubusercontent.com/3046459/134042147-6069bf81-eea7-4572-b093-0323eb073932.png)

I'm not sure what the `candidateFilterSearch` query arguments are meant to be, but the query needs to return something.

Reply:
Yea that was from me testing something, forgot to remove that.